src/os/FileJournal.cc

```diff
@@ -1242,7 +1249,7 @@ void FileJournal::write_thread_entry()
     }
     
 #ifdef HAVE_LIBAIO
-    if (aio) {
+    if (aio && !g_conf->journal_dynamic_throttle) {
```

Comment:
I'm not sure.  Early in newstore I was having trouble with a small aio 
queue size and had to add retries.  Now I can't trigger EAGAIN from 
io_submit even with a small queue, and I'm not sure what changed.  If we 
do we will trigger this usleep

https://github.com/ceph/ceph/blob/master/src/os/filestore/FileJournal.cc#L1470

which is mostly harmless.

I would expect this would trigger much with the new throttle sleeps, 
though.. does it actually slow things down?